spec/integration/resource/api300/c7000/switch/update_spec.rb

```diff
@@ -4,6 +4,7 @@
 RSpec.describe klass, integration: true, type: UPDATE do
   let(:current_client) { $client_300 }
   include_examples 'SwitchUpdateExample', 'integration api300 context'
```

Comment:
This `include_examples 'SwitchUpdateExample', 'integration api300 context'` could be insert into '`SwitchUpdateExample  API300`', so the test of api300 would call the tests of api200, once the all api200 tests works to api300 in this case.
But it is just a observation, do not need do that.